tools/arriba.wdl

```diff
@@ -0,0 +1,315 @@
+## [Homepage](https://arriba.readthedocs.io/en/latest/)
+#
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
+version 1.1
+
+task arriba {
+    meta {
+        description: "Run Arriba structural variant caller on a STAR-produced BAM file."
+        outputs: {
+            fusions: "Output file of fusions in TSV format"
+            discarded_fusions: "Output file of discarded fusions in TSV format"
+        }
+    }
+
+    parameter_meta {
+        bam: "Input BAM format file on which to call fusions"
+        gtf: "GTF features file. Gzipped or uncompressed."
+        reference_fasta_gz: "Gzipped reference genome in FASTA format"
+        chimeric_sam: "Optional input file of chimeric reads in SAM format, from older versions of STAR"
+        exclude_list: "Optional input file of regions to exclude from analysis in tab delimited format"
+        known_fusions: "Optional input file of known fusions in tab delimited format"
+        annotate_fusions: "Optional input file in tab delimited format of fusions to annotate with tags"
+        protein_domains: "Optional input file of protein domains coordinates in GFF3 format"
+        wgs_svs: "Optional input file of structural variants found by WGS in tab delimited or VCF format"
```

Comment:
Can we get a link to how these should be formatted? I think just saying `in tab delimited format` is not enough. Reading that, **I** do not know how to write/supply these extra files. So we need more detail or instruction. But I don't want these to become super-long because it's detailing a whole file specification. That would be a pain to write and a pain to read.

I think adding `external_help` to each of these would be good. Would it be weird if all of them have the same link? Maybe instead, add `See 'meta.external_help' for more information.` (and obviously put a link at that location)

Reply:
Some of these (`exclude_list`, `known_fusions`, `protein_domains`) are included in the Arriba release. The others are optional.